core/src/main/java/com/cosium/spring/data/jpa/entity/graph/repository/EntityGraphJpaSpecificationExecutor.java

```diff
@@ -30,4 +31,6 @@
 
   /** @see JpaSpecificationExecutor#findAll(Specification, Sort) */
   List<T> findAll(Specification<T> spec, Sort sort, EntityGraph entityGraph);
+
+  Slice<T> findSlice(Specification<T> spec, Pageable pageable, EntityGraph entityGraph);
```

Comment:
Does this method minus the entity graph exists on spring-data-jpa?
If not, there is no reason to add it to this extension. People are free to extend the library with their own methods without needing to patch this extension.

Same remarks for the other method.

Reply:
ok, I will close this PR then.